packages/loading-indicator/src/BlobLoader/BlobLoader.spec.tsx

```diff
@@ -1,9 +1,15 @@
 import React from 'react';
 import { render } from '@testing-library/react';
 import { axe } from 'jest-axe';
+import { setupJestCanvasMock } from 'jest-canvas-mock';
 
 import BlobLoader from './BlobLoader';
 
+// this could be setup globally in jest.config, but done in this .spec file to pass depcheck
```

Comment:
Since jest-canvas-mock was previously setup globally, depcheck thought the dependency wasn't being used anywhere and raised an error. I figured it would be better to just add it here since it's accurate to say that package is only being used in loading-indicator.